### PR TITLE
vnet_edge quick fix

### PR DESCRIPTION
**Pull request by akry**

### CODE REVIEW

File: vnet/lib/vnet/core/translations/vnet_edge_handler.rb [hunk 1]
```diff
@@ -193,14 +204,15 @@ def handle_packet_from_edge_port(message)
...
-                          }, {})
+      # flows << Flow.create(TABLE_EDGE_DST, 2, {
+      #                      :eth_dst => src_mac,
+      #                      :metadata => METADATA_TYPE_VIRTUAL_TO_EDGE,
+      #                      :metadata_mask => METADATA_TYPE_MASK
+      #                     }, {
+      #                      :mod_vlan_vid => vlan_vid,
+      #                      :output => message.in_port
+      #                     }, {})
+      `ovs-ofctl -O OpenFlow13 add-flow br0 'table=#{TABLE_EDGE_DST} priority=2 eth_dst=#{src_mac} metadata=#{METADATA_TYPE_VIRTUAL_TO_EDGE}/#{METADATA_TYPE_MASK} actions=mod_vlan_vid:#{vlan_vid},output:#{message.in_port}'`
```
Comment (hansode) [inline vnet/lib/vnet/core/translations/vnet_edge_handler.rb]: __[reek]__
* [DuplicateMethodCall] calls message.in_port 2 times

File: vnet/lib/vnet/core/translations/vnet_edge_handler.rb [hunk 2]
```diff
@@ -127,20 +134,24 @@ def handle_packet_from_host_port(message)
...
+        #                      :eth_type => 0x0806,
+        #                      :eth_dst => dst_mac,
+        #                      :metadata => METADATA_TYPE_VIRTUAL_TO_EDGE,
+        #                      :metadata_mask => METADATA_TYPE_MASK
+        #                     }, {:eth_dst => MAC_BROADCAST}.merge(actions), {})
+        option_str=""
+        case actions
+        when Array
+          actions.each do |action|
+            option_str << "mod_vlan_vid:#{action[:mod_vlan_vid]}," if action[:mod_vlan_vid]
```
Comment (hansode) [inline vnet/lib/vnet/core/translations/vnet_edge_handler.rb]: __[reek]__
* [DuplicateMethodCall] calls action[:mod_vlan_vid] 2 times
* [DuplicateMethodCall] calls action[:mod_vlan_vid] 2 times

File: vnet/lib/vnet/core/translations/vnet_edge_handler.rb [hunk 3]
```diff
@@ -127,20 +134,24 @@ def handle_packet_from_host_port(message)
...
+            option_str << "output:#{action[:output]}," if action[:output]
```
Comment (hansode) [inline vnet/lib/vnet/core/translations/vnet_edge_handler.rb]: __[reek]__
* [DuplicateMethodCall] calls action[:output] 2 times
* [DuplicateMethodCall] calls action[:output] 2 times

File: vnet/lib/vnet/core/translations/vnet_edge_handler.rb [hunk 4]
```diff
@@ -118,6 +118,13 @@ def handle_packet_from_host_port(message)
                            :metadata => METADATA_TYPE_VIRTUAL_TO_EDGE,
                            :metadata_mask => METADATA_TYPE_MASK
                           })
+      flows << Flow.create(TABLE_EDGE_DST, 2, {
+                           :eth_dst => src_mac,
+                           :metadata => METADATA_TYPE_EDGE_TO_VIRTUAL,
+                           :metadata_mask => METADATA_TYPE_MASK
+                          }, {
+                           :output => message.in_port
```
Comment (hansode) [inline vnet/lib/vnet/core/translations/vnet_edge_handler.rb]: __[reek]__
* [DuplicateMethodCall] calls message.in_port 2 times

**Comment by Metallion** [comment]
I think the reeks are ok. While it's true that we shouldn't do multiple hash lookups, I feel like it's still within acceptable bounds here.

**Comment by Metallion** [comment]
The edge integration test still fails.

```
19:04:40 --------------------------------------------------
19:04:40 simple: success
19:04:40 router_v2v: success
19:04:40 router_p2v: success
19:04:40 event: success
19:04:40 service: success
19:04:40 secg: success
19:04:40 secg_reference: success
19:04:40 edge: failure
19:04:40 --------------------------------------------------
```

**Comment by Metallion** [comment]
The first time in a loooong time the integration test is 100% green. :) You have no idea how big a smile I have on my face right now. Merging. :+1:

**Comment by YamazakiYasuhiro** [comment]
```
                                                     ..::''''::..
                                           .:::.   .;''        ``;.
   ....                                    :::::  ::    ::  ::    ::
 ,;' .;:                ()  ..:            `:::' ::     ::  ::     ::
 ::.      ..:,:;.,:;.    .   ::   .::::.    `:'  :: .:' ::  :: `:. ::
  '''::,   ::  ::  ::  `::   ::  ;:   .::    :   ::  :          :  ::
,:';  ::;  ::  ::  ::   ::   ::  ::,::''.    .    :: `:.      .:' ::
`:,,,,;;' ,;; ,;;, ;;, ,;;, ,;;, `:,,,,:'   :;:    `;..``::::''..;'
                                                     ``::,,,,::''
```

**Pull request merged**